### PR TITLE
Add Feedza to ember users

### PR DESCRIPTION
**Pull request by raikoroje**

### CODE REVIEW

File: data/users.yml [hunk 1]
```diff
@@ -783,6 +783,10 @@
   image: ibotta.png
   alt: "Ibotta"
   use: "We use Ember for building our internal and external web applications."
+- url: https://feedza.com/
+  image: feedza.png
+  alt: "Feedza"
+  use: "The feedza web app was build with Ember"
```
Comment (acorncom) [inline data/users.yml]: _was built_

**Comment by acorncom** [comment]
@wifelette ?

**Comment by raikoroje** [comment]
Hey guys, any updates on this? :)

**Comment by wifelette** [comment]
Sorry, no idea how, but I lost this in the fray :p All good now!

**Pull request merged**